src/Plugin/Block/EcFooterBlock.php

```diff
@@ -5,58 +5,19 @@
 namespace Drupal\oe_corporate_blocks\Plugin\Block;
 
 use Drupal\Component\Utility\NestedArray;
-use Drupal\Core\Block\BlockBase;
 use Drupal\Core\Cache\CacheableMetadata;
-use Drupal\Core\Config\ConfigFactoryInterface;
 use Drupal\Core\Plugin\ContainerFactoryPluginInterface;
-use Symfony\Component\DependencyInjection\ContainerInterface;
 
 /**
  * Provides the corporate footer block.
  *
  * @Block(
  *   id = "oe_footer",
```

Comment:
Let's use this occasion to properly prefix plugin names with the module's name, let's change this one into

```
oe_corporate_blocks_ec_footer
```